build_env.rst

```diff
@@ -24,13 +24,13 @@ Cache Folders
 {Singularity} will cache SIF container images generated from remote
 sources, and any OCI/docker layers used to create them. The cache is
 created at ``$HOME/.singularity/cache`` by default. The location of the
-cache can be changed by setting the ``SINGULARITY_CACHEDIR`` environment
+cache can be changed using the ``SINGULARITY_CACHEDIR`` environment
```

Comment:
Personally prefer 'setting' here. If someone searches for 'setting environment variable' then it's more likely to give them what they need to know.